pkg/agent/baker_test.go

```diff
@@ -281,6 +367,86 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 				TransparentHugePageDefrag:  "defer+madvise",
 				SwapFileSizeMB:             &swapFileSizeMB,
 			}
+		}),
+		Entry("AKSWindows2019 with k8s version 1.16", "AKSWindows2019+K8S116", "1.16.15", func(config *datamodel.NodeBootstrappingConfiguration) {
```

Comment:
I think a better way is to separate windows and linux test cases so you don't have to touch linux. It will make future work and test easier

Reply:
there is not much we gain by mixing them together

Reply:
I agree with you and not.
1. It indeed makes test complex, but it may in some way protect both linux and windows case. What if we touched something  that will change Linux or Windows behaviour that we cannot see from the testdata? Linux and windows share many function/variables